sqlglot/parser.py

```diff
@@ -2659,7 +2659,7 @@ def parse_conditional_insert() -> t.Optional[exp.ConditionalInsert]:
         )
 
     def _parse_insert(self) -> t.Union[exp.Insert, exp.MultitableInserts]:
-        comments = ensure_list(self._prev_comments)
+        comments = []
```

Comment:
Looks like the other statements completely removed their `comments`, did we have to preserve this one for some reason?

Reply:
Yes, we're adding comments to the insert that aren't in the INSERT token.